components/discount/registerDiscount.tsx

```diff
@@ -370,6 +425,17 @@ const RegisterDiscount: FunctionComponent<RegisterDiscountProps> = ({
               isSwissResident={isSwissResident}
```

Comment:
<img width="663" alt="Capture d’écran 2024-05-23 à 15 19 20" src="https://github.com/starknet-id/app.starknet.id/assets/78437165/a6fde126-1017-49d9-9b17-c667fcd9a12c">

In the process of trying to make things as simple as possible, I would like to delete both of these `TextField ` and `isSwissForm` and all the associated metadata ...

Considering this it might be better to rewrite a component `freeRegister.tsx` considering we'll have a lot of changes I think it would be cleaner overall

Reply:
<img width="612" alt="Capture d’écran 2024-05-23 à 15 22 31" src="https://github.com/starknet-id/app.starknet.id/assets/78437165/8a663761-151c-4ac6-8162-5488df17a87d">

In the same context we would like to delete this option and the call data associated